recipes/yacman/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "yacman" %}
+{% set version = "0.6.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "37cd49463880f4c9cfee0dc8280bc893fc3bf392615d12b35c97609def71071c"
+
+build:
+  number: 0
```

I think this recipe could be ``noarch``
```suggestion
  number: 0
  noarch: python
```